### DISCUSSION TITLE
Which certificate store is used by `CefSharp`?

### DISCUSSION THREAD

**Discussion by brycx** [cefsharp/CefSharp#4467] [Q&A] [answered]
It wasn't clear to me whether `CefSharp` uses the Chromium certificate store (from CEF) for trusted root certificates or the machine store, e.g Windows certificate store?

Apologies in advance if this is the wrong place for this question.

**Comment by amaitland** [comment, marked as answer]
`CefSharp` uses the `CEF` default when it comes to choosing a certificate store. You can manually control the behaviour (until `Chromium` removes the option) using the **ChromeRootStoreUsed** feature.

- https://chromium.googlesource.com/chromium/src/+/main/net/data/ssl/chrome_root_store/faq.md#given-the-rollout-is-gradual_how-can-i-tell-if-these-features-are-in-use-on-my-system
- https://chromium.googlesource.com/chromium/src/+/main/net/data/ssl/chrome_root_store/faq.md#how-can-i-tell-which-certificates-are-trusted-by-the-chrome-root-store
- https://chromium.googlesource.com/chromium/src/+/main/net/data/ssl/chrome_root_store/testing.md

**Reply by brycx** [reply]
Thank you!
